src/apis/stockx/asks/index.js

```diff
@@ -1,7 +1,8 @@
 import moment from 'moment';
 
 import Base from '../../base';
-import { randomInclusive, errors } from '../../../utils';
+import { randomInclusive } from '../../../utils/index';
+import { checkStatus } from '../../../utils/errors';
```

Comment:
You’ll probably have to change this import here as well.
```suggestion
import { checkStatus } from '../../../utils';
```